packages/grafana-ui/src/components/BarChart/bars.ts

```diff
@@ -1,13 +1,10 @@
-import uPlot, { Axis, Series, Cursor, Select } from 'uplot';
+import uPlot, { Axis, Series } from 'uplot';
 import { Quadtree, Rect, pointWithin } from './quadtree';
 import { distribute, SPACE_BETWEEN } from './distribute';
+import { TooltipInterpolator } from '../uPlot/types';
+import { ScaleDirection, ScaleOrientation } from '../uPlot/config';
 
-const pxRatio = devicePixelRatio;
-
-const groupDistr = SPACE_BETWEEN;
-const barDistr = SPACE_BETWEEN;
```

Comment:
not sure why these need to be removed. the whole point was to make it easy to change in one place.

Reply:
SPACE_BETWEEN has no semantic meaning, but barDistr does. it's the same reason you may want `declare type Spacing: number`. even though you can simply use `: number` inline everywhere.